libs/application/ui-fields/src/lib/FindVehicleFormField/FindVehicleFormField.tsx

```diff
@@ -63,14 +66,18 @@ const isVehicleType = function <T>(
   )
 }
 
+export const formatIsk = (value: number): string =>
+  value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, '.') + ' kr.'
+
```

Comment:
We have a shared `formatCurrency` function that I would prefer you'd use for consistency

Reply:
Good pointer, will replace with this one